ourplays/urls.py

```diff
@@ -1,4 +1,7 @@
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
+    path('user/',include('user.urls')),
+    path('place/',include('place.urls')),
+    path('reservation/',include('reservation.urls')),
```

Comment:
```suggestion
    path('user',include('user.urls')),
    path('place',include('place.urls')),
    path('reservation',include('reservation.urls')),
```
main url은 슬래시 없이 선언하시는 것이 위코드 컨벤션입니다 :)